resources/lib/library.py

```diff
@@ -1494,8 +1494,8 @@ def explorer( self, history, location, label, thumbnail, itemType, isWidget = Fa
                     listitem.setProperty( "widgetPath", location )
 
                 elif itemType == "32011" or itemType == "32019" or itemType == "32073":
-                    action = 'ActivateWindow(10501,"' + location + '",return)'
-                    listitem.setProperty( "windowID", "10501" )
+                    action = 'ActivateWindow(10502,"' + location + '",return)'
+                    listitem.setProperty( "windowID", "10502" )
```

Comment:
As this is a quick fix, I'll leave it as-is for now and rework to window aliases when I have a bit more time. Thanks as always for the comments :)